aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -277,7 +281,7 @@ struct ReduceOp {
   InputCalculator input_calc;
   OutputCalculator output_calc;
   const void* src;
-  void* dst;
+  const char* dst[2]; //TODO: make it accept more then two destinations
```

Comment:
If we are not actually planning to do this, remove the TODO.

If we are, we should make an issue and reference it in the comment.

Reply:
Changed